### PR TITLE
Petitioner and Respondent Screens

### PR DESCRIPTION
**Pull request by sophiedrake**
Petitioner and Respondent screens, currently these are duplicated even though they ask for the same information on for both petitioner and respondent, can be re-factored at a later date.

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -95,21 +95,47 @@ router.post('/v1/screening-questions/respondent-address', function(req, res) {
...
+  req.session.petitioner.town = req.body.town;
+  req.session.petitioner.county = req.body.county;
+  req.session.petitioner.postcode = req.body.postcode;
+
+  req.session.petitioner.address = [
+    req.body.street1,
+    req.body.street2,
+    req.body.town,
+    req.body.county
+  ].filter(function (val) {return val;}).join(', ');
```
Comment (anperez78) [inline app/routes.js]: Is this filter doing any filtering ?

File: app/routes.js [hunk 2]
```diff
@@ -135,52 +161,78 @@ router.get('/v1/about-you/address/select', function(req, res) {
...
+  req.session.respondent.town = req.body.town;
+  req.session.respondent.county = req.body.county;
+  req.session.respondent.postcode = req.body.postcode;
+
+  req.session.respondent.address = [
+    req.body.street1,
+    req.body.street2,
+    req.body.town,
+    req.body.county
+  ].filter(function (val) {return val;}).join(', ');
```
Comment (anperez78) [inline app/routes.js]: same comment as above

**Comment by anperez78** [comment]
Reviewed. Some comments left.

File: app/routes.js (hunk 1 above)
Comment (sophiedrake) [inline app/routes.js]: yeah it removes the ones that haven't been filled in 'undefined'

File: app/routes.js (hunk 2 above)
Comment (sophiedrake) [inline app/routes.js]: same as above

File: app/routes.js (hunk 1 above)
Comment (anperez78) [inline app/routes.js]: nice! 👍

**Comment by anperez78** [comment]
👍

**Pull request merged**